.github/workflows/update-wiki.yml

```
@@ -0,0 +1,92 @@
+name: Update Wiki
+
+on:
```

yamllint

[warning] truthy value should be one of [false, true]

(truthy)